pom.xml

```diff
@@ -12,6 +12,7 @@
   <packaging>pom</packaging>
 
   <properties>
+    <nuxeo.jsf.version>11.1-SNAPSHOT</nuxeo.jsf.version>
```

Comment:
What's the point?

Reply:
To define and use a property for all dependency management to not rely on `${project.version}`.